nf_core/list.py

```diff
@@ -60,19 +60,15 @@ def get_remote_workflows(self):
 
         # List all repositories at nf-core
         logging.debug("Fetching list of nf-core workflows")
-        gh_api_url = 'https://api.github.com/orgs/nf-core/repos?per_page=100'
-        response = requests.get(gh_api_url, timeout=10)
+        #gh_api_url = 'https://api.github.com/orgs/nf-core/repos?per_page=100'
+        nfcore_url = 'http://nf-co.re/pipelines.json'
```

Comment:
It won't do anything yet, but do you think it's worth adding metadata to the URL here? eg.
```
http://nf-co.re/pipelines.json?nft_ver=1.0&cmd=list
```
_(other nf-core tools also use this class, so we should probably pass the command name through as a variable)_


Then at some point we can convert the nf-co.re web page to a PHP script instead of a static JSON file and collect stats on what's pinging it.

Reply:
NB: If we want to collect these stats, we should probably remove all of the `requests_cache` code. To be honest, I wouldn't be sad to see that go anyway as it has caused some problems and complicates the code (eg. it creates a bunch of files on the user's system and stuff). The only reason I added it was to avoid the GitHub API limiting problem.

Reply:
> It won't do anything yet, but do you think it's worth adding metadata to the URL here? eg.

Actually, I don't have an opinion about this yet. Usually I would avoid adding complexity until it is really necessary or the benefit significant. Maybe we can let this idea incubating or have some more discussion about it in the gitter?